packages/components/src/styles/hidden-links/hidden-links.stories.ts

```diff
@@ -0,0 +1,192 @@
+  argTypes
+};
+
+/**
+ * Per default the hidden links are optimized for a single `<sd-button>`.
+ */
+export const Default = {
+  args: overrideArgs({
+    type: 'slot',
+    name: 'default',
+    value: html`<sd-button href="#">Skip to content</sd-button>`
+  }),
+  render: (args: any) => {
+    return generateTemplate({
+      options: {
+        templateContent: html`<div class="relative bg-primary-100 w-full h-[256px] p-8">
+          <div class="%CLASSES%">%SLOT%</div>
+          <p>Tab into this area to show a single button.</p>
+        </div>`
+      },
+      args
+    });
+  }
+};
+
+/**
+ * The style appends its position in regard of the parent container's position. It can be used with any element.
+ */
+export const SurroundingContent = {
+  render: () => {
+    return html` <div style="height: 100px; --sd-header-inner-max-width: 1456px;">
+      <style>
+        .sb-main-padded.sb-show-main {
+          padding: 0;
+        }
+        :root {
+          --sd-header-padding: 8px 16px;
+        }
+
+        sd-header {
+          height: 64px;
+          max-height: 140px;
+        }
+
+        @media (min-width: 376px) {
+          :root {
+            --sd-header-padding: 24px;
+          }
+          sd-header {
+            height: 96px;
+          }
+        }
+
+        @media (min-width: 1025px) {
+          :root {
+            --sd-header-padding: 24px 32px 0 32px;
+          }
+          sd-header {
+            height: 140px;
+          }
+        }
+
+        @media (min-width: 1440px) {
+          :root {
+            --sd-header-padding: 24px 48px 0 48px;
+          }
+        }
+      </style>
+      <div class="z-20 sd-hidden-links sd-hidden-links--multiple">
+        <sd-navigation-item href="#">Suche</sd-navigation-item>
+        <sd-navigation-item href="#">Inhalt</sd-navigation-item>
+        <sd-navigation-item href="#">Fußbereich</sd-navigation-item>
+      </div>
+      <sd-header class="z-10" fixed>
+        <div class="flex justify-between items-center">
+          <!-- top-left-area start !-->
+          <a class="flex flex-shrink" href="#">
+            <img class="h-8 md:h-12 lg:h-[56px]" src="images/logo-unioninvestment-lg.svg" alt="Logo" />
+          </a>
+        </div>
+      </sd-header>
+    </div>`;
+  }
+};
+
+/**
+ * You can "stack" multiple `sd-hidden-links` by adding multiple of them to the same parent. This is recommended for 2 links.
+ */
+export const StackLinks = {
+  render: () =>
+    html`<div class="relative bg-primary-100 w-full h-[256px] p-8">
+      <div class="sd-hidden-links"><sd-button href="#">Skip to Search</sd-button></div>
+      <div class="sd-hidden-links"><sd-button href="#">Skip to Content</sd-button></div>
+      <p>Tab through this area to see buttons one after another.</p>
+    </div>`
+};
+
+/**
+ * Use the `sd-hidden-links--multiple` modifier to show mutiple `<sd-navigation-item>` elements. This is recommended for >2 links.
```

Comment:
* Use the `sd-hidden-links--multiple` modifier to show mutiple `<sd-navigation-item>` elements. (Recommended for >2 links)